src/hotspot/os/linux/procMapsParser.inline.hpp

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
Is it important to have these in an inline file? They're very small, seems like they can just be in the header.